operator/v1/types_network.go

```diff
@@ -209,3 +283,29 @@ const (
 	// for sophisticated network isolation and segmenting. This is the default.
 	SDNModeNetworkPolicy SDNMode = "NetworkPolicy"
 )
+
+// MacvlanMode is the Mode of macvlan. The value are lowercase to match the CNI plugin
+// config values. See "man ip-link" for its detail.
+type MacvlanMode string
+
+const (
+	// MacvlanModeBridge is the macvlan with thin bridge function.
+	MacvlanModeBridge MacvlanMode = "bridge"
```

Comment:
"Bridge" - all constants should be FirstLetterCaps

Reply:
changed.